python/lsst/ctrl/oods/butlerProxy.py

```diff
@@ -30,7 +30,7 @@ class ButlerProxy(object):
     butlerConfig: `dict`
         details on how to construct and configure the butler
     """
-    def __init__(self, butlerConfig):
+    def __init__(self, butlerConfig, publisher=None, publisher_queue=None):
```

Comment:
Though it is not shown in the diff, but it looks like you forgot to document newly added arguments in the class docstring.

Reply:
Fixed